### PR TITLE
Comments section

### LINKED ISSUE #2449: Comments section
Design: https://www.figma.com/file/sMydmZJlzhK91FTIPOd7D2/Video-page?node-id=2555%3A452099

**AC:**

- [ ] Loading state
  - [ ] Sorting disabled 
  - [ ] 4 comments in loading state
- [ ] Comments section disabled state (https://www.figma.com/file/sMydmZJlzhK91FTIPOd7D2/Video-page?node-id=2411%3A219280)
- [ ] Data fetching (we can do mocked data if QN not available yet)
- [ ] Display all top-level comments by default
- [ ] Handle guest user https://www.figma.com/file/sMydmZJlzhK91FTIPOd7D2/Video-page?node-id=2431%3A214814

**Out of scope:**
- Sorting (#2442)

### PR DESCRIPTION
**Pull request by rafalpawlow**
Closes #2449 
I didn't implement comment input since it's still in review https://github.com/Joystream/atlas/pull/2558

**Design**
https://www.figma.com/file/sMydmZJlzhK91FTIPOd7D2/Video-page?node-id=2213%3A140537

### CODE REVIEW

**Comment by rafalpawlow** [comment]
Waiting for https://github.com/Joystream/atlas/pull/2474

File: packages/atlas/src/views/viewer/VideoView/CommentsSection.tsx [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+        <Select
+          size="small"
+          labelPosition="left"
+          label={mdMatch ? 'Sort by' : ''}
+          value={sortCommentsBy}
+          items={COMMENTS_SORT_OPTIONS}
+          onChange={handleSorting}
+          disabled={loading}
+        />
+      </CommentsSectionHeader>
```
Comment (drillprop) [inline packages/atlas/src/views/viewer/VideoView/CommentsSection.tsx:62]: You didn't add a divider below the header.

image

**Review by drillprop — CHANGES_REQUESTED** [review]
Found just one minor thing, besides looks and works awesome.

**Review by drillprop — APPROVED** [review]

**Comment by kdembler** [comment]
Nice!

**Pull request merged**